lib/core/src/display/rendering/camera.rs

```diff
@@ -1,38 +1,111 @@
 use crate::prelude::*;
```

Comment:
A side note here - let's leave this camera implementation for now, but on my branch, I've got much more complex one, so we will be merging it soon.